account_onboarder/main.py

```diff
@@ -253,8 +277,11 @@ def _load_config(config_file):
     :param config_file: the file path
     :return: Loaded config
     """
-    with open(config_file, "r") as fs:
-        return yaml.safe_load(fs)
+    try:
+        with open(config_file, "r") as fs:
+            return yaml.safe_load(fs)
+    except yaml.YAMLError as e:
+        return e
```

Comment:
this could be raised as an exception instead and catch at the config error below (wrapping the check in the try block so all the sys exit codes originate and are accordingly classified within the try/except block).

Reply:
Fixed